package.json

```diff
@@ -23,4 +24,4 @@
     "lint-staged": "^13.2.2",
     "quicktype": "^23.0.48"
   }
-}
+}
```

Comment:
I'm curious about the cause of this. Which IDE and OS are you using? https://github.com/configu/configu/pull/167#discussion_r1268777475

Reply:
VSCode with Linux (Ubuntu) :+1: